kumascript/macros/GlossarySidebar.ejs

```diff
@@ -0,0 +1,32 @@
+<%
+var locale = env.locale;
+var baseURL = '/' + locale + '/docs/Glossary';
+
+var text = mdn.localStringMap({
+  'en-US': {
+    'Glossary_home_page' : 'MDN Web Docs Glossary',
+  },
+  'es': {
+    'Glossary_home_page' : 'Glosario de MDN Web Docs',
+  },
+  'fr': {
+    'Glossary_home_page' : "Glossaire MDN Web Docs",
+  },
+  'ja': {
+    'Glossary_home_page' : 'MDN Web Docs 用語集',
+  },
+  'ko': {
+    'Glossary_home_page' : 'MDN Web Docs 용어집',
+  },
+  'ru': {
+    'Glossary_home_page' : 'Глоссарий MDN Web Docs',
+  }
+});
```

Comment:
For a single key, you can just `mdn.localString()` as well:

```suggestion
var text = mdn.localString({
  'en-US': 'MDN Web Docs Glossary',
  'es': 'Glosario de MDN Web Docs',
  'fr': "Glossaire MDN Web Docs",
  'ja': 'MDN Web Docs 用語集',
  'ko': 'MDN Web Docs 용어집',
  'ru': 'Глоссарий MDN Web Docs',
});
```

**But** the best way is to just reuse the `short-title` or `title` from the Glossary index page directly, to avoid translations inside the macro altogether:

Reply:
Very good, going with `short-title`

Reply:
See https://github.com/mdn/yari/pull/8997/commits/880c89fb50d5cc88adfaa0dcd6ae7527827f910e